gitlab/src/main/java/cz/xtf/gitlab/GitLabUtil.java

```diff
@@ -0,0 +1,296 @@
+package cz.xtf.gitlab;
+
+import org.apache.commons.io.FileUtils;
+
+import org.gitlab.api.GitlabAPI;
+import org.gitlab.api.models.GitlabGroup;
+import org.gitlab.api.models.GitlabProject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Random;
+import java.util.UUID;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+import cz.xtf.core.openshift.OpenShifts;
+import cz.xtf.gitlab.config.GitLabConfig;
+
+public class GitLabUtil {
+
+	private static final String SEPARATOR = "------";
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(GitLabUtil.class);
```

Comment:
We should use Lomboks `@Slf4j` tag to use consistent logging approach across the project.